From eac2cd4ed64e471b15d0b75ab7c01b8082ea5cf1 Mon Sep 17 00:00:00 2001
From: Helge Deller <deller@gmx.de>
Date: Fri, 31 Jul 2026 14:56:51 +0200
Subject: [PATCH 10613/15333] UAPI: Drop PER_HPUX personality

Two and a half decades ago, there were plans to support HP-UX syscalls
in Linux. This was never fully implemented, never worked and was never
actually used by anybody.
Let's simply drop this define now and free it up for other usages.

Reviewed-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Helge Deller <deller@gmx.de>
---
 include/uapi/linux/personality.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/uapi/linux/personality.h b/include/uapi/linux/personality.h
index 49796b7756af..243f5f80a7b8 100644
--- a/include/uapi/linux/personality.h
+++ b/include/uapi/linux/personality.h
@@ -62,7 +62,6 @@ enum {
 	PER_SOLARIS =		0x000d | STICKY_TIMEOUTS,
 	PER_UW7 =		0x000e | STICKY_TIMEOUTS | MMAP_PAGE_ZERO,
 	PER_OSF4 =		0x000f,			 /* OSF/1 v4 */
-	PER_HPUX =		0x0010,
 	PER_MASK =		0x00ff,
 };
 
-- 
2.55.0

